Index: src/components/Common/Containers/RoundedContainer.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import styled from \"styled-components\";\n\ntype RoundedContainerProps = {\n  children: React.ReactNode;\n  theme?: \"light\" | \"dark\";\n};\n\nconst colors = {\n  light: \"#ffffff\",\n  dark: \"#000000\",\n};\n\nexport const RoundedContainer: React.FC<RoundedContainerProps> = ({\n  children,\n  theme = \"light\",\n}) => {\n  return (\n    <RoundedContainerDiv\n      style={{\n        backgroundColor: colors[theme],\n      }}\n    >\n      {children}\n    </RoundedContainerDiv>\n  );\n};\n\nconst RoundedContainerDiv = styled.div`\n  border-radius: 24px;\n  padding: 16px;\n`;\n
===================================================================
diff --git a/src/components/Common/Containers/RoundedContainer.tsx b/src/components/Common/Containers/RoundedContainer.tsx
--- a/src/components/Common/Containers/RoundedContainer.tsx	
+++ b/src/components/Common/Containers/RoundedContainer.tsx	
@@ -3,6 +3,8 @@
 type RoundedContainerProps = {
   children: React.ReactNode;
   theme?: "light" | "dark";
+  size?: string;
+
 };
 
 const colors = {
@@ -13,11 +15,13 @@
 export const RoundedContainer: React.FC<RoundedContainerProps> = ({
   children,
   theme = "light",
+  size = "100%"
 }) => {
   return (
     <RoundedContainerDiv
       style={{
         backgroundColor: colors[theme],
+        width: size,
       }}
     >
       {children}
@@ -28,4 +32,5 @@
 const RoundedContainerDiv = styled.div`
   border-radius: 24px;
   padding: 16px;
+
 `;
