Index: src/themes/dark/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { Theme } from \"@typesDef/themes\";\nimport { createColor } from \"@utils/createColor\";\n\nconst dark: Theme = {\n  id: 0,\n  name: \"dark\",\n  colors: {\n    primary: \"#90e0ef\",\n    secondary: \"#caf0f8\",\n\n    background: {\n      primary: \"linear-gradient(123deg, #2B2D42 0%, #023E8A 50%, #2B2D42 100%)\",\n      secondary: createColor(30, 31, 32, 0.2),\n      contrast: \"\",\n    },\n    text: {\n      primary: \"#2b2d42\",\n      secondary: createColor(141, 149, 157),\n      contrast: \"#f8f9fa\",\n    },\n    button: {\n      background: \"\",\n      text: \"\",\n      outline: \"\",\n    },\n  },\n};\n\nexport default dark;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/themes/dark/index.ts b/src/themes/dark/index.ts
--- a/src/themes/dark/index.ts	
+++ b/src/themes/dark/index.ts	
@@ -9,7 +9,7 @@
     secondary: "#caf0f8",
 
     background: {
-      primary: "linear-gradient(123deg, #2B2D42 0%, #023E8A 50%, #2B2D42 100%)",
+      primary: "#F5F6F7",
       secondary: createColor(30, 31, 32, 0.2),
       contrast: "",
     },
Index: src/themes/light/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { Theme } from \"@typesDef/themes\";\nimport { createColor } from \"@utils/createColor\";\n\nconst light: Theme = {\n  id: 0,\n  name: \"light\",\n  colors: {\n    primary: \"#00b4d8\",\n    secondary: \"#90e0ef\",\n\n    background: {\n      primary:\n        \"linear-gradient(123deg, #FFF 23.25%, #C5F1FB 48.67%, #F1F6F6 76.07%)\",\n      secondary: createColor(246, 6, 123, 0.2),\n      contrast: \"\",\n    },\n    text: {\n      primary: \"#2b2d42\",\n      secondary: createColor(141, 149, 157),\n      contrast: \"#2b2d42\",\n    },\n    button: {\n      background: \"\",\n      text: \"\",\n      outline: \"\",\n    },\n  },\n};\n\nexport default light;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/themes/light/index.ts b/src/themes/light/index.ts
--- a/src/themes/light/index.ts	
+++ b/src/themes/light/index.ts	
@@ -10,7 +10,7 @@
 
     background: {
       primary:
-        "linear-gradient(123deg, #FFF 23.25%, #C5F1FB 48.67%, #F1F6F6 76.07%)",
+        "#F5F6F7",
       secondary: createColor(246, 6, 123, 0.2),
       contrast: "",
     },
Index: src/pages/index.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { HomeScreen } from \"@screens/Homepage\";\n\nexport default function Home() {\n  // ! Delete the HomeScreen component to remove the landing page\n  return <HomeScreen />;\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/index.tsx b/src/pages/index.tsx
--- a/src/pages/index.tsx	
+++ b/src/pages/index.tsx	
@@ -1,6 +1,9 @@
-import { HomeScreen } from "@screens/Homepage";
+import {Header} from "@components/Header/Index"
+
 
 export default function Home() {
   // ! Delete the HomeScreen component to remove the landing page
-  return <HomeScreen />;
+ return <>
+   <Header />
+ </>
 }
